### DISCUSSION TITLE
Adding custom difficulties?

### DISCUSSION THREAD

**Discussion by EcoVeni** [KadeArchive/Kade-Engine#2374] [Q&A]
hi! i am currently making a mod and i really want to make a new difficulty called: "Hell" but when i looked up on the source code guide it was outdated beacuse i didn't have the lines of code that the guide was telling me to do even tho this is a fresh new copy. Can someone tell me how to do it? i am using kade engine 1.6

**Comment by libracrumbs** [comment]
AtlasSux did a tutorial on that. https://github.com/KadeDev/Kade-Engine/discussions/2113
